recipes/reno-sd/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
This file is in the tarball. You do not need to add it again here.

Reply:
Removed!